lib/cards/__tests__/components/CardsVisualTest.vue

```diff
@@ -0,0 +1,90 @@
+<template>
```

Comment:
Great! Will this handle https://github.com/learningequality/kolibri-design-system/issues/889 then right?